Build failure: timeout in python312Packages.eventlet (0.37.0)

## Steps To Reproduce

1. build python312Packages.eventlet

## Build log

<details>

<summary>Build Log</summary>

```
=================================== FAILURES ===================================
______________ TestHttpd.test_server_connection_timeout_exception ______________
...
E       tests.TestIsTakingTooLong: 5

eventlet/greenthread.py:346: TestIsTakingTooLong
=============================== warnings summary ===============================
```

</details>

## Metadata

doesn't matter

## Notify maintainers

@SuperSandro2000 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/eventlet/default.nix
+++ b/pkgs/development/python-modules/eventlet/default.nix
@@ -68,6 +68,8 @@ buildPythonPackage rec {
     # Tests requires network access
     "test_getaddrinfo"
     "test_hosts_no_network"
+    # flaky test, depends on builder performance
+    "test_server_connection_timeout_exception"
   ];
 
   pythonImportsCheck = [ "eventlet" ];

